src/checkpoint.hh

```diff
@@ -994,7 +1149,7 @@ SUPPORTED_EXTERNAL_TYPES = {"Rope", "answer_pknot_mfe", "pktype",
      stream << "double get_tabulated_vals_percentage() const {" << endl;
      inc_indent();
      stream << indent() << "return static_cast<double>(tabulated_vals_counter) "
-            << "/ tabulated.size() * 100.0;" << endl;
```

Comment:
can we format the percentage values to "%.2f", I feel `69.4831%` is too precise, better make it `69.48%` and then more it thus consistent: `Table is 0% filled.`--> `Table is 0.00% filled.`
Btw: I really love this feature :-)

Reply:
Sure.

 I think it's a cool feature as well. Had fun implementing this and learnt a lot!